docs/pages/management/operations/db-ca-migrations.mdx

```diff
@@ -0,0 +1,182 @@
+---
+title: Database CA Migrations
+description: How to complete Teleport Database CA migrations.
+---
+
+In Teleport Database Access, self-hosted databases must be configured with 
+certificates to enable mTLS authentication via the Teleport Database Service.
+
+Teleport 15 introduced a new `db_client` certificate authority (CA) to split the
+responsibilities of the Teleport `db` CA, which was acting as both host and 
+client CA for Teleport self-hosted database access.
+
+Teleport's host/client database CA split is intended to limit the potential for
+lateral movement to other resources in the event that a database instance's
+private key is compromised.
+
+The `db` and `db_client` CAs were both introduced as an automatic migration
+after upgrading to Teleport 10 and Teleport 15, respectively.
+
+This guide will explain why these CAs were created and how to determine if your
+CA(s) should be rotated to complete the migration process.
+
+## Prerequisites
+
+(!docs/pages/includes/edition-prereqs-tabs.mdx!)
+
+- (!docs/pages/includes/tctl.mdx!)
+
+- A Teleport cluster that was created prior to Teleport 15.
+  If your Teleport cluster was created with Teleport 15+, then this guide does
```

Comment:
Would it make sense to include this guide in older versions of the docs, but not v15?

Reply:
I think we should keep it around for a while at least, hard to say.
Technically it's useful info so long as we have customers who haven't rotated their database CAs since upgrading to v10 or v15.